src/portal_visualization/builders/base_builders.py

```diff
@@ -11,7 +11,7 @@ def __init__(self, entity, groups_token, assets_endpoint, **kwargs):
         # Just so it has the same signature as the other builders
         pass
 
-    def get_conf_cells(self):
+    def get_conf_cells(self, **kwargs):
```

Comment:
Not sure about this... It would be a surprise if kwargs were supplied, so probably better to revert this?